subversion.yaml

```diff
@@ -21,6 +21,7 @@ environment:
       - libmagic-dev
       - libtool
       - lz4-dev
+      - serf-dev
```

Comment:
needs epoch bump for the package to rebuild with the changes

Reply:
epoch is bumped now.